grafana/overlays/moc/smaug/dashboards/cluster-management/cluster-storage-overview.yaml

```diff
@@ -11,143 +11,774 @@ spec:
+                "type": "value"
+              }
+            ],
+            "thresholds": {
+              "mode": "absolute",
+              "steps": [
+                {
+                  "color": "#760613",
+                  "value": null
+                },
+                {
+                  "color": "dark-red",
+                  "value": 1e-8
+                },
+                {
+                  "color": "semi-dark-red",
+                  "value": 0.05
+                },
+                {
+                  "color": "red",
+                  "value": 0.25
+                },
+                {
+                  "color": "light-orange",
+                  "value": 0.4
+                },
+                {
+                  "color": "yellow",
+                  "value": 0.5
+                },
+                {
+                  "color": "light-green",
+                  "value": 0.65
+                },
+                {
+                  "color": "semi-dark-green",
+                  "value": 0.85
+                }
+              ]
+            },
+            "unit": "percentunit"
+          },
+          "overrides": [
+            {
+              "matcher": {
+                "id": "byName",
+                "options": "Namespace"
+              },
+              "properties": [
+                {
+                  "id": "color",
+                  "value": {
+                    "fixedColor": "text",
+                    "mode": "fixed"
+                  }
+                }
+              ]
+            },
+            {
+              "matcher": {
+                "id": "byName",
+                "options": "Persistent Volume Claim (PVC)"
+              },
+              "properties": [
+                {
+                  "id": "color",
+                  "value": {
+                    "fixedColor": "text",
+                    "mode": "fixed"
+                  }
+                }
+              ]
+            }
+          ]
+        },
+        "gridPos": {
+          "h": 12,
+          "w": 12,
+          "x": 0,
+          "y": 10
+        },
+        "id": 22,
+        "options": {
+          "footer": {
+            "fields": "",
+            "reducer": [
+              "sum"
+            ],
+            "show": false
+          },
+          "showHeader": true
+        },
+        "pluginVersion": "8.4.3",
+        "targets": [
+          {
+            "datasource": {
+              "type": "prometheus",
+              "uid": "PF0BA5EC94FCB9BBE"
+            },
+            "exemplar": false,
+            "expr": "bottomk(20, kubelet_volume_stats_available_bytes{cluster=\"$cluster\"} / kubelet_volume_stats_capacity_bytes{cluster=\"$cluster\"})",
+            "format": "table",
+            "instant": true,
+            "interval": "",
+            "legendFormat": "{{namespace}}",
+            "refId": "A"
+          }
+        ],
+        "title": "Worst 20 PVCs by capacity remaining",
```

Comment:
> "Worst 20 PVCs by capacity remaining"

"Wrost" feels like a reprimand and implies a fully/mostly used up PVC is bad. Can we use different wording here?

Reply:
Hey @HumairAK that is a great point, I was very much on the fence about the title. On the latest commit I resolved your other comments, but I also changed the title of this panel to "Lowest 20 PVCs by capacity remaining" let me know what you think.